README.md

```diff
@@ -29,15 +27,18 @@ The backend (API) code is stored in the `server/` directory.
 - [Wiki](https://github.com/ProgramEquity/amplify/wiki): API methods, Data Structures
 
 **Resources:**
-- We meet every **Wednesday from 12-1 pm pt and Thursdays from 8-8:30 pm pt**. Sign up for an [orientation](https://forms.gle/4miQJ8ccuWdeJha16)
+- We meet every **Wednesday from 12-1 p.m. PT and Thursdays from 8-8:30 p.m. PT**. Sign up for an [orientation](https://forms.gle/4miQJ8ccuWdeJha16)
 - Try out our [demo](https://www.figma.com/file/46c9cmuTiCpFA4DHB8OK0H/Amplify-User-Interface-%2B-Design-Guide?node-id=1585%3A653) or review [App Research](https://www.notion.so/programequity/Dare-to-Dream-Civic-Engagement-is-key-to-change-595ca4db3a2948c6b44569b58d530c8c)
 
 
 
-## Project setup
+# Project setup
+In order to get started, you can clone, download a ZIP, or fork this repository to work on your local machine. If you would like to get started with Codespaces instead, the video below will walk you through setting up your Codespace.
 
 https://user-images.githubusercontent.com/9143339/159093687-6fc90733-0599-445c-b08b-a6378d988e4b.mov
 
+<figcaption align = "center"><b>Codespaces Set Up</b></figcaption>
```

Comment:
I don't think centring works here in GitHub's Markdown (check the preview), so you can get rid of the `figcaption` tag.

```suggestion
<b>Codespaces Set Up</b>
```